Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreterEmitter.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2024 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/PowerPC/CachedInterpreter/CachedInterpreterEmitter.h"
+
+#include <algorithm>
+#include <cstring>
+
+#include "Common/Assert.h"
+#include "Common/MsgHandler.h"
+
+void CachedInterpreterEmitter::Write(AnyCallback callback, const void* operands, std::size_t size)
+{
+  DEBUG_ASSERT(reinterpret_cast<std::uintptr_t>(m_code) % alignof(AnyCallback) == 0);
+  if (m_code + sizeof(callback) + size >= m_code_end)
+  {
+    m_write_failed = true;
+    return;
+  }
+  std::memcpy(m_code, &callback, sizeof(callback));
+  m_code += sizeof(callback);
+  std::memcpy(m_code, operands, size);
+  m_code += size;
+}
+
+s32 CachedInterpreterEmitter::PoisonCallback(PowerPC::PowerPCState& ppc_state, const void* operands)
+{
+  PanicAlertFmtT("The Cached Interpreter reached a poisoned callback. This should never happen!");
```

Comment:
```suggestion
  PanicAlertFmt("The Cached Interpreter reached a poisoned callback. This should never happen!");
```

Let's avoid unnecessary work for the translators. (Users won't understand what this means anyway. This is the kind of message that would also work well as an assert.)

Reply:
I can switch it to a DYNA_REC ASSERT_MSG.